src/Service/Rekognition/tests/Unit/Input/RecognizeCelebritiesRequestTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AsyncAws\Rekognition\Tests\Unit\Input;
+
+use AsyncAws\Core\Test\TestCase;
+use AsyncAws\Rekognition\Input\RecognizeCelebritiesRequest;
+use AsyncAws\Rekognition\ValueObject\Image;
+use AsyncAws\Rekognition\ValueObject\S3Object;
+
+class RecognizeCelebritiesRequestTest extends TestCase
+{
+    public function testRequest(): void
+    {
+        $input = new RecognizeCelebritiesRequest([
+            'Image' => new Image([
+                'Bytes' => 'change me',
+                'S3Object' => new S3Object([
+                    'Bucket' => 'change me',
+                    'Name' => 'change me',
+                    'Version' => 'change me',
```

Comment:
Same here, let's use values that look more like real values.

Reply:
done